Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js

```diff
@@ -1190,6 +1190,7 @@ localizedStrings["Observer Callback"] = "Observer Callback";
 localizedStrings["Observer Handlers:"] = "Observer Handlers:";
 localizedStrings["Observers:"] = "Observers:";
 localizedStrings["Off"] = "Off";
+localizedStrings["On"] = "On";
```

Comment:
NIT: Can you please add a comment to help translators to identify what "On" represents?

The translation can vary according to context.
Thinking in Romanian here, the former is more common, but the later is more suitable for a switch:

On = Pornit | Activ
Off = Oprit | Inactiv

I'm sure that's the case for other languages too.

Reply:
Good point, thanks! Fixed now.